geoportailv3/externs/fuse.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Externs for fuse.js
+ * @see https://github.com/krisk/Fuse/blob/master/README.md
+ * @externs
+ */
+
+/**
+ * @typedef {{
+ *    keys: (Array.<string>),
+ *    id: (string|undefined),
+ *    threshold: (number|undefined),
+ *    distance: (number|undefined),
+ *    includeScore: (boolean|undefined)
+ * }}
+ */
+var FuseOptions;
```

Comment:
I don't think a `@typedef` is sufficient. If these properties, or just some of them, are renamed by the compiler then it does mean a `@typedef` is not sufficient. In that case, this is what needs to be written:

```js
/**
 * @typedef {{
 *   keys: Array.<string>,
 *   ...
 * }}
 */
FuseOptions;

/**
 * @type {Array.<string>}
 */
FuseOptions.keys;

// same for every other properties…
```